code-samples/additional-features/cooldowns/commands/moderation/prune.js

```diff
@@ -0,0 +1,22 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+	cooldown: 5,
+	data: new SlashCommandBuilder()
+		.setName('prune')
+		.setDescription('Prune up to 99 messages.')
+		.addIntegerOption(option => option.setName('amount').setDescription('Number of messages to prune')),
+	async execute(interaction) {
+		const amount = interaction.options.getInteger('amount');
+
+		if (amount < 1 || amount > 99) {
+			return interaction.reply({ content: 'You need to input a number between 1 and 99.', ephemeral: true });
```

Comment:
```suggestion
		.addIntegerOption(option =>
            option.setName("amount").setDescription("Number of messages to prune").setMinValue(1).setMaxValue(99)
         ),
	async execute(interaction) {
		const amount = interaction.options.getInteger('amount');
```

Why not use `.setMinValue()` and `.setMaxValue()` here? This might need correct formatting though